packages/web-components/src/accordion/accordion-item/accordion-item.options.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * An Accordion Item can be medium, large, and larger in height
+ */
+export type AccordionItemSize = 'medium' | 'large' | 'larger';
```

Comment:
For all the options, export const object + type, see `text` component as an example:
https://github.com/microsoft/fluentui/blob/5e3ba35835c0a5487b574ea58a51cccd67b5fa8c/packages/web-components/src/text/text.options.ts#L31-L41